modules/python/src2/gen2.py

```diff
@@ -246,9 +247,14 @@ def __init__(self, decl):
 
 class ClassInfo(object):
     def __init__(self, name, decl=None):
+        # Scope name can be a module or other class e.g. cv::SimpleBlobDetector::Params
+        self.scope_name, self.original_name = name.rsplit(".", 1)
+
+        self.export_name = self.original_name
+        self.scope_name = re.sub(r"^cv\.?", "", self.scope_name)
```

Comment:
Current code emits (`pyopencv_generated_modules_content.h`):

```
static PyMethodDef methods_viz[] = {
    {"PyAffine3d_Identity", CV_PY_FN_WITH_KW_(pyopencv_cv_viz_PyAffine3d_Identity, 0), "PyAffine3d_Identity() -> retval\n."},
    ...
```

1. "PyAffine3d_Identity" -> "Affine3d_Identity" as we see and use `Affine3d` from Python.
2. `pyopencv_cv_viz_PyAffine3d_Identity`. With similar case in Java it is important to have "wrapper" name in method declaration, e.g: `pyopencv_cv_viz_Affine3d_Identity()`
3. Doc string should be replaced too.

```
struct CV_EXPORTS_W_SIMPLE CV_WRAP_AS(Affine3d) PyAffine3d
                                         ^          ^
                we must see this in binding         |
                                                  we use this internally in C++ code only
```